src/utils/components/ConfirmActionMessage/ConfirmActionMessage.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { Trans } from 'react-i18next';
+
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { K8sResourceCommon } from '@openshift-console/dynamic-plugin-sdk';
+
+const ConfirmActionMessage: React.FC<{ obj: K8sResourceCommon; action?: string }> = ({
+  obj,
+  action,
+}) => {
+  const { t } = useKubevirtTranslation();
+  const objNamespace = obj?.metadata?.namespace;
+
+  return (
+    <Trans t={t}>
+      Are you sure you want to {action || 'delete'} <strong>{obj?.metadata?.name}</strong>
```

Comment:
Sorry, I missed this before. You can set 'delete' as the default value in the function arguments list. That would get rid of the `|| 'delete'` on this line.

Reply:
Yeah, thanks, sometimes there are more options how to implement something :)

Reply:
Agreed :)